guides/v2.1/frontend-dev-guide/css-topics/css-overview.md

```diff
@@ -32,6 +32,8 @@ There are a couple options to help with CSS and site performance.
 
 * Minification of CSS files reduces the file size being sent. It does this by stripping white space within the file.
 
+* Make sure that you set your Magento application to the developer mode because merge CSS option not available in production mode, after merging CSS you can set developer to production mode.
```

Comment:
Make sure that Magento is set to `developer` mode. The `merge CSS` option is not available in `production` mode. After merging your CSS, you can set Magento back to `production`.

Reply:
> Make sure that Magento is set to `developer` mode. The `merge CSS` option is not available in `production` mode. After merging your CSS, you can set Magento back to `production`.

Yes, @dobooth after `merge CSS` we can set Magento back to `production`.

Reply:
I was fixing the text for you. Please update the PR.